spec/features/persisting_browser_locale_after_sign_in_spec.rb

```diff
@@ -12,8 +12,13 @@
       'your mental health.'
     )
 
+    open_header_if_hidden
+
     within('#header') { find('a[href="/users/sign_out"]').click }
-    expect(find('#header')).to have_content('SIGN IN')
+
+    open_header_if_hidden
+
+    expect(find('#header')).to have_content(/sign in/i)
```

Comment:
Tthis text is capitalized when shown in the header and titleized when accessed through the hamburger menu, this searches via a case-insensitive regex to find the the link to sign in